src/Tempest/Auth/composer.json

```
@@ -0,0 +1,12 @@
+{
+    "name": "tempest/auth",
+    "require": {
```

You'd probable have a better description than this 😅 

```suggestion
    "name": "tempest/auth",
    "description": "Tempest authentication",
    "require": {
```